doc/setup-binderhub.rst

```diff
@@ -1,104 +1,140 @@
 Set up BinderHub
 ================
 
-.. note::
-
-   If you wish to use encryption with ``letsencrypt``, BinderHub requires
-   that you configure the DNS of a web address to point to the BinderHub IP
-   address. This means that you need to have control over a URL such as
-   ``www.mydomain.com``.
-
-Download the Helm Chart and deployment repo for BinderHub
----------------------------------------------------------
-
 BinderHub uses Helm Charts to set up the applications we'll use in our Binder
 deployment. If you're curious about what Helm Charts are and how they're
 used here, see the `Zero to JupyterHub guide
 <https://zero-to-jupyterhub.readthedocs.io/en/latest/tools.html#helm>`_.
 
-First grab the latest helm chart for BinderHub and install it.::
-
-    helm repo add jupyterhub https://jupyterhub.github.io/helm-chart
-    helm install --name binder jupyterhub/binderhub --version v0.1.0-464c4f5
-
-
-Configure DNS
--------------
-
-Next we'll configure our web address to point to the IP address for our
-BinderHub.
+Below we'll cover how to configure your Helm Chart, and how to create your
+BinderHub deployment.
 
-Find the ``EXTERNAL-IP`` of the **nginx-ingress-controller**::
+Configure the helm chart
```

Comment:
I think this should be perhaps 'creating configuration files' or 'installing binderhub'. The current title seems super generic.

Reply:
Perhaps just don't have a heading here?

Reply:
How about "prepare information for the helm chart"?

Reply:
How about "preparing to install"? But there's no actions you are performing in this subheading right?

Reply:
I like that one - and you're generating the tokens and "making sure" you still have that JSON file around :-)